### DISCUSSION TITLE
Does no GUI mean incorrect offsets?

### DISCUSSION THREAD

**Discussion by hey101** [bckd00r/LViewPlus#29] [Q&A]
*removed*

**Comment by itvhot1** [comment]
Hi ! i am Ve from DIscord sever.

**Comment by bckd00r** [comment]
> Hello, I had no issues getting everything compiled and set up. I thought I extracted the new offsets correctly for 11.16 and updated them in the offsets file, but upon loading into a game, I have no GUI or any drawings. From what I have read in various threads, it seems that I still have incorrect offsets. I just want to make sure that is the issue or is there something else I need to try and do? I'll go ahead and attempt to extract the offsets again in case I messed up. Also if the offsets being incorrect causes no GUI to appear, what are the bare minimum ones needed to get the GUI to appear? Or is it a case where if I get one offset wrong then it just will not work? I would rather find the offsets on my own as I want to learn the whole process but I feel I have managed to get a few incorrectly and want to learn from my mistakes.
> 
> BTW thank you for this amazing project, I am learning so much exploring the code behind it all!

for python you have to install some packages

**Reply by hey101** [reply]
*removed*

**Reply by bckd00r** [reply]
maybe you can check again or ask in the forum, normally it is forbidden to ask such questions in places like github or discord otherwise my or your account may be banned :(

compile the project as x86 and try again with the addresses I will send

`auto GameTime = 0x2FC1988;
auto LocalPlayer = 0x2FC90D8;
auto Minimap = 0x2FE844C;
auto MinimapObject = 0x2FC969C;
auto ObjectManager = 0x172C9E4;
auto UnderMouseObject = 0x2FC9AB8;
auto Renderer = 0x2FF092C;
auto ZoomClass = 0x2FBEAE8;
auto ViewMatrix = 0x2FEDAC0;`

This offsets from UC i don't know if these are working and i have an internal project right now so i don't use these addresses normally.

**Reply by hey101** [reply]
I removed my prior question and comments and deleted their history. Didn't realize that was forbidden to ask that. ill try with the new info you gave and report back. Thanks!

**Reply by bckd00r** [reply]
you don't need to remove it because as a result I am helping here

**Reply by hey101** [reply]
Well, it was incorrect offsets causing the drawings to not show up, I had the renderer one incorrect. Now I just need to fix the drawings for 4k at 200% scale so they are put in the correct place. So now the fun can begin. Thank you for taking the time to help me out. Best of luck with your internal project!

**Reply by 7ORAS99** [reply]
pls i neeed the new offests

**Comment by lOster231** [comment]
same error since 11.17

**Comment by 7ORAS99** [comment]
![فقف](https://user-images.githubusercontent.com/92491575/137229355-7828e691-67cb-47c8-a74f-9e235fa276ca.png)
Can someone help me because there is nothing happening when I enter the training just this is what it appears to m

**Reply by 7ORAS99** [reply]
loading addon activator and loading addon ashews not appers just match to be won found
